src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -698,6 +707,14 @@
                         });
                     }
 
+                    /* Prevents from disallowing clicks on "copy to clipboard" elements inside toggles */
+                    var copyToClipboardElements = toggles[i].querySelectorAll('span[data-clipboard-text]')
```

Comment:
Why using `for (...)` here and `.forEach` at line 40.
Same for `addEventListener(element` vs `element.addEventListener`)

Reply:
It would be even better to use event delegation instead.

Reply:
I think that this code is actually useless because the click listener for toggle it is not attached to span elements. What do you think?